docs/user/tutorials/01-40-expose-workload/v2alpha1/01-43-expose-workload-short-host-name.md

```diff
@@ -0,0 +1,97 @@
+# Expose a Workload with Short Host Name
+
+This tutorial demonstrates how to expose an unsecured instance of the HTTPBin Service using a short host name instead of the full domain name. Using a short host makes it simpler to apply APIRules because the domain name is automatically retrieved from the referenced Gateway, and you don’t have to manually set it in each APIRule. This might be particularly useful when reconfiguring resources in a new cluster, as it reduces the chance of errors and streamlines the process.
+
+> [!WARNING]
+>  Exposing a workload to the outside world is a potential security vulnerability, so tread carefully. In a production environment, always secure the workload you expose with [JWT](../../01-50-expose-and-secure-a-workload/v2alpha1/01-52-expose-and-secure-workload-jwt.md).
+
+## Prerequisites
+
+* [Deploy a sample HTTPBin Service](../../01-00-create-workload.md).
+* [Set up your custom domain](../../01-10-setup-custom-domain-for-workload.md) or use a Kyma domain instead.
+
+## Steps
+
+### Expose Your Workload
+
+1. Depending on whether you use your custom domain or a Kyma domain, export the necessary values as environment variables:
+
+  <!-- tabs:start -->
+  #### **Custom Domain**
+
+  ```bash
+  export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+  export GATEWAY=$NAMESPACE/httpbin-gateway
+  ```
+  #### **Kyma Domain**
+
+  ```bash
+  export DOMAIN_TO_EXPOSE_WORKLOADS={KYMA_DOMAIN_NAME}
+  export GATEWAY=kyma-system/kyma-gateway
+  ```
+  <!-- tabs:end -->
+
+2. To expose an instance of the HTTPBin Service, create the following APIRule:
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: gateway.kyma-project.io/v2alpha1
+    kind: APIRule
+    metadata:
+      name: httpbin
+      namespace: $NAMESPACE
+    spec:
+      hosts:
+        - httpbin
+      service:
+        name: $SERVICE_NAME
+        namespace: $NAMESPACE
+        port: 8000
+      gateway: $GATEWAY
+      rules:
+        - path: /*
+          methods: ["GET"]
+          noAuth: true
+        - path: /post
+          methods: ["POST"]
+          noAuth: true
+    EOF
+    ```
+
```

Comment:
```suggestion
    
    The host domain name will be obtained from the referenced Gateway.
```
it's standard information about the tutorial, so I don't think we need to include it in a note